apps/web/pages/[user]/[type].tsx

```diff
@@ -197,34 +244,58 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   const schedule = eventType.schedule
     ? { ...eventType.schedule }
     : {
-        ...user.schedules.filter(
-          (schedule) => !user.defaultScheduleId || schedule.id === user.defaultScheduleId
+        ...users[0].schedules.filter(
+          (schedule) => !users[0].defaultScheduleId || schedule.id === users[0].defaultScheduleId
         )[0],
       };
 
-  const timeZone = schedule.timeZone || eventType.timeZone || user.timeZone;
+  const timeZone =
+    users.length > 1 ? undefined : schedule.timeZone || eventType.timeZone || users[0].timeZone;
 
   const workingHours = getWorkingHours(
     {
       timeZone,
     },
-    schedule.availability || (eventType.availability.length ? eventType.availability : user.availability)
+    users.length > 1
+      ? eventType.availability || undefined
+      : schedule.availability ||
+          (eventType.availability.length ? eventType.availability : users[0].availability)
   );
-
   eventTypeObject.schedule = null;
   eventTypeObject.availability = [];
 
+  const isDynamicGroupBooking = users.length > 1 ? true : false;
+
+  const profile =
+    users.length > 1
+      ? {
+          name: getGroupName(usernameList),
+          image: null,
+          slug: typeParam,
+          theme: null,
+          weekStart: "Sunday",
+          brandColor: "",
+          darkBrandColor: "",
+          allowDynamicBooking: users.some((user) => {
+            return !user.allowDynamicBooking;
+          })
+            ? false
+            : true,
+        }
```

Comment:
Checking if any of the users have disallowed dynamic group booking. Only considered in case of dynamic group booking